internal/addon/addon.go

```diff
@@ -1,12 +1,23 @@
 package addon
 
 import (
+	"errors"
+	"fmt"
+
+	otelv1alpha1 "github.com/open-telemetry/opentelemetry-operator/apis/v1alpha1"
+	loggingv1 "github.com/openshift/cluster-logging-operator/apis/logging/v1"
 	"open-cluster-management.io/addon-framework/pkg/agent"
 	"open-cluster-management.io/addon-framework/pkg/utils"
 	addonapiv1alpha1 "open-cluster-management.io/api/addon/v1alpha1"
+	workapiv1 "open-cluster-management.io/api/work/v1"
```

Comment:
nit. the k8s api convention aliasing is `packagename+version`
```suggestion
	workv1 "open-cluster-management.io/api/work/v1"
```